sdk/go/common/apitype/core.go

```diff
@@ -411,6 +413,16 @@ const (
 	VCSRepositoryRootTag StackTagName = "vcs:root"
 )
 
+// ConfigKey is the key for a configuration value. This is just a string, but we use a type alias to provide a richer
+// description of how the string is used in our apitype definitions.
+type ConfigKey = string
```

Comment:
I'm not sure I would bother with this, you can still have doc comment on the const string below and we'r not using this type anywhere.